.github/workflows/main.yml

```diff
@@ -14,7 +14,7 @@ concurrency:
 
 jobs:
   lint:
-    runs-on: ubuntu-22.04-16core
+    runs-on: ubuntu-22.04-2core
```

Comment:
@RenfengLiu : is this the right way to change to a 2 core runner? where can i see all the available options? thanks :)

Reply:
updated the yml file as suggested, now the ci is back to normal, thanks!